gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValue.java

```diff
@@ -427,6 +539,60 @@ public void setGradoopId(GradoopId gradoopIdValue) {
     Bytes.putBytes(rawBytes, OFFSET, valueBytes, 0, valueBytes.length);
   }
 
+  /**
+   * Sets the wrapped value as {@code Map} value.
+   *
+   * @param map value
+   */
+  public void setMap(Map<PropertyValue, PropertyValue> map) {
+    int size =
+      map.keySet().stream().mapToInt(PropertyValue::byteSize).sum() +
+      map.keySet().size() * Integer.BYTES +
```

Comment:
i think these two `map.keySet().size() * Integer.BYTES +` are not necessary anymore. I think this reserves too much?